bin/heudiconv

```diff
@@ -142,92 +283,133 @@ def find_files(regex, topdir=curdir, exclude=None, exclude_vcs=True, dirs=False)
 find_files.__doc__ %= (_VCS_REGEX,)
 
 
-def process_dicoms(fl, dcmsession=None, basedir=None, dcmfilter=None):
-    """Process list of dicoms and return  seqinfo and file group
+def group_dicoms_into_seqinfos(fl, flfilter=None, dcmfilter=None, grouping='studyUID'):
+    """Process list of dicoms and return seqinfo and file group
+
+    `seqinfo` contains per-sequence extract of fields from DICOMs which
+    will be later provided into heuristics to decide on filenames
 
     Parameters
     ----------
     fl : list of str
```

Comment:
I see no reason for type (`_list`) suffix really -- making it `files` (noone works with open files in here, so no fear of confusing with file instances) and `flfilter` into `file_filter`